### PR TITLE
Fix some typos in comments

### PR DESCRIPTION
**Pull request by Tim-Tech-Dev**
Fix some typos and grammar in comments from core and dashboard.

Also removed some abbreviations

### CODE REVIEW

**Comment by Tim-Tech-Dev** [comment]
### Two caveats:

https://github.com/Tim-Tech-Dev/nodecg-io/blob/40de57966186dbbe2552d1d9e51943dc1a91987a/nodecg-io-core/extension/bundleManager.ts#L81-L83

Should that be:
```
        // Get the service dependency that manages dependants for this service type
```
Or:
```
        // Get the service dependency that manages dependence for this service type
```
Or something else?

---

https://github.com/Tim-Tech-Dev/nodecg-io/blob/40de57966186dbbe2552d1d9e51943dc1a91987a/nodecg-io-core/extension/persistenceManager.ts#L304-L309

There seems to be missing a noun.

**Comment by daniel0611** [comment]
1. that should be `dependence`, I think
2. the `the` in the second comment line shouldn't be there

File: nodecg-io-core/dashboard/crypto.ts [hunk 1]
```diff
@@ -83,15 +85,15 @@ export async function sendAuthenticatedMessage<V>(messageName: string, message:
 }
 
 /**
- * Returns whether a password has been set in the crypto module aka. whether is is authenticated.
+ * Returns whether a password has been set in the crypto module aka. whether is authenticated.
```
Comment (daniel0611) [inline nodecg-io-core/dashboard/crypto.ts:88]: The first `is` was most likely an typo of `it`.
```suggestion
 * Returns whether a password has been set in the crypto module aka. whether it is authenticated.
```

File: nodecg-io-core/dashboard/esbuild.config.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ const BuildOptions = {
     entryPoints: entryPoints,
     /**
      * This sets the output format for the generated JavaScript files. We are
-     * using th `iife`, witch format stands for "immediately-invoked function
+     * using the `iife`, witch format stands for "immediately-invoked function
```
Comment (daniel0611) [inline nodecg-io-core/dashboard/esbuild.config.js:51]:
```suggestion
     * using the `iife`, which format stands for "immediately-invoked function
```

File: services/nodecg-io-debug/esbuild.config.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ const BuildOptions = {
     entryPoints: entryPoints,
     /**
      * This sets the output format for the generated JavaScript files. We are
-     * using th `iife`, witch format stands for "immediately-invoked function
+     * using the `iife`, witch format stands for "immediately-invoked function
```
Comment (daniel0611) [inline services/nodecg-io-debug/esbuild.config.js:51]:
```suggestion
     * using the `iife`, which format stands for "immediately-invoked function
```

**Review by daniel0611 — APPROVED** [review]
Thanks

**Pull request merged**